components/tracker/src/lib.rs

```diff
@@ -34,9 +34,38 @@ impl Tracker {
     pub fn write_scan_detail(&self, detail_v2: &mut pb::ScanDetailV2) {
         detail_v2.set_rocksdb_block_read_byte(self.metrics.block_read_byte);
         detail_v2.set_rocksdb_block_read_count(self.metrics.block_read_count);
+        detail_v2.set_rocksdb_block_read_nanos(self.metrics.block_read_nanos);
```

Comment:
Also need to update `metrics.block_read_nanos` when reporting metrics to the tracker?